### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by SaneMyburg**
Dynamically added a popup window for both mobile and desktop versions.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -363,6 +363,74 @@ h6 {
...
+  height: 90%;
+  background: #fff;
+  overflow: scroll;
+  border-radius: 20px;
+  position: relative;
+  padding-left: 15px;
+  display: none;
+  margin: 75px;
+  z-index: 99999;
+}
```
Comment (codepantha) [inline style.css]: - [x] The mobile menu modal currently looks too squished together and is a stark contrast to the given Figma design. 
Please reduce the `75px` margin that you specified on both sides. That is the main culprit 👍 I believe you will have a much more beautiful pop-up window if you simply adjust your left and right margins 😎 
###  Figma Design
image

### Your Implementation
image

File: style.css [hunk 2]
```diff
@@ -670,6 +772,49 @@ h6 {
...
+    display: flex;
+    list-style: none;
+    gap: 2%;
+  }
+
+  .modal-img {
+    width: 90%;
+    margin-left: 5%;
+    margin-top: 2%;
+  }
```
Comment (codepantha) [inline style.css]: - [x] Please adjust your margin left and right to match the specified margins in the Figma template you chose. 
- [x] Please note the "See live" and "See source" buttons below. Kindly make sure that they look like the ones in the Figma design 👍 Kindly see the images below
## Figma Design
image

## Your Implementation
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRES CHANGES ♻️ 

## Hi team 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ No linter errors
✔️ Used GitHub Flow

## Required Changes ♻️
_Check the comments under the review._

- [x] Since it's a collaborative project, kindly include the name of your coding partner in the "Authors" section of your README 👍 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- 

- [x] Please consider ensuring that you and your partner contribute equally whenever you have a pair-programming project. This will help you understand how to work collaboratively and would be hugely beneficial for your career as a software developer.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @SaneMyburg, and Team👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**